server/core/runtime/post_workflow_hook_runner.go

```diff
@@ -52,7 +52,8 @@ func (wh DefaultPostWorkflowHookRunner) Run(ctx models.WorkflowHookCommandContex
 	cmd.Env = finalEnvVars
 	out, err := cmd.CombinedOutput()
 
-	wh.OutputHandler.SendWorkflowHook(ctx, string(out), false)
+	outString := strings.ReplaceAll(string(out), "\n", "\r\n")
+	wh.OutputHandler.SendWorkflowHook(ctx, outString, false)
 	wh.OutputHandler.SendWorkflowHook(ctx, "\n", true)
 
 	if err != nil {
```

Comment:
In this `err` block, why not return `outString` instead of `string(out)` ? This way we do not have to change all the `ExpOut` in the tests

Reply:
I didn't want to touch the output that was being returned from the function, as I don't know what it may be used for.

Reply:
cc @GenPage @jamengual thoughts on this?

Reply:
The output of the function is being ignored using the blank identifier, so currently serves no purpose. https://github.com/runatlantis/atlantis/blob/4eee2e9e0dd8ce99bd120d5b4f7bbd1c298542e3/server/events/pre_workflow_hooks_command_runner.go#L118